src/metabase/driver/postgres.clj

```diff
@@ -296,6 +300,57 @@
   [driver [_ arg]]
   (sql.qp/->honeysql driver [:percentile arg 0.5]))
 
+(defmethod sql.qp/->honeysql [:postgres :datetimediff]
+  [driver [_ x y unit]]
+  (let [x (sql.qp/->honeysql driver x)
+        y (sql.qp/->honeysql driver y)
+        helper
```

Comment:
should we move this out to an actual `defn`?

Should at least be more readable and hopefully easier to have a docstring.

Reply:
https://github.com/metabase/metabase/commit/f4077e325f2991cf151d2e261655a17601a25d19